frontend/app/src/components/ExternalTradeForm.vue

```diff
@@ -141,6 +141,33 @@
                 <v-icon>mdi-swap-vertical</v-icon>
               </v-btn>
             </div>
+            <div v-if="shouldRenderSummary" class="text-caption green--text mt-n5">
+              <v-icon small class="mr-2 green--text">
+                mdi-comment-quote
+              </v-icon>
+              <template v-if="type === 'buy'">
+                <strong>{{ $t('external_trade_form.summary.label') }}</strong>
+                {{ $t('external_trade_form.summary.buy') }}
+                <strong>{{ amount }} {{ getAssetText(base) }}</strong>
+                {{ $t('external_trade_form.summary.with') }}
+                <strong>{{ getAssetText(quote) }}</strong>
+                {{ $t('external_trade_form.summary.for') }}
+                <strong>{{ rate }} {{ getAssetText(quote) }}</strong>
+                {{ $t('external_trade_form.summary.per') }}
+                <strong>{{ getAssetText(base) }}</strong>
+              </template>
+              <template v-else-if="type === 'sell'">
+                <strong>{{ $t('external_trade_form.summary.label') }}</strong>
+                {{ $t('external_trade_form.summary.sell') }}
+                <strong>{{ amount }} {{ getAssetText(base) }}</strong>
+                {{ $t('external_trade_form.summary.with') }}
+                <strong>{{ getAssetText(quote) }}</strong>
+                {{ $t('external_trade_form.summary.for') }}
```

Comment:
maybe this would be a nice use case for https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage

Something like this:
https://github.com/rotki/rotki/blob/0a5dfde83557a97ea6d6d5f1798bcd77dcff5aaa/frontend/app/src/components/history/UpgradeRow.vue#L7

Reply:
This one is updated @kelsos